src/styles/product-card.scss

```diff
@@ -0,0 +1,93 @@
+.product-card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  transition: 300ms;
+}
+
+.product-card:hover {
+  transform: scale(1.2);
+}
+
+// .product-card:hover .product-card__product-name {
+//   color: #34568b;
+// }
+
+.product-card p {
```

Comment:
you may reset it for `p` tag generally, inside a separate file (`_normalize.css`, etc.)